compiler/damlc/tests/daml-test-files/ScenarioError.daml

```diff
@@ -1,5 +1,6 @@
 -- This tests that defining a scenario triggers an error when scenarios are disabled.
 -- @ERROR Scenarios are no longer supported.
+-- Remove when scenarios are fully deleted (TODO - make/find a ticket)
```

Comment:
```suggestion
-- TODO https://github.com/digital-asset/daml/issues/16711
--   Remove when scenarios are fully deleted
```

Reply:
Is this the right ticket? I dont think this relates to the canton work

Reply:
Going to link to [16583](https://github.com/digital-asset/daml/issues/16583)

Reply:
My bad, 16583 is the good one.